### PR TITLE
Student Information Mangement

### PR DESCRIPTION
**Pull request by katieann7**

### CODE REVIEW

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
remove integrated from the project name

**Comment by katieann7** [comment]
Done po sir

File: student-information-management-application/build.xml [hunk 1]
```diff
@@ -37,4 +37,5 @@
         </javadoc>
         <echo message="java doc has been generated!" />
```
Comment (RoCS2024) [inline student-information-management-application/build.xml:38]: Check project name

File: student-information-management-application/ivy.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<ivy-module version="2.0"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:noNamespaceSchemaLocation="http://ant.apache.org/ivy/schemas/ivy.xsd">
+    <info
+            organisation="com.rogationist.college"
+            module="ivy-test"
+            status="integration">
+    </info>
+    <dependencies>
+        <dependency org="com.oracle.database.jdbc" name="ojdbc11" rev="23.3.0.23.09"/>
```
Comment (RoCS2024) [inline student-information-management-application/ivy.xml:11]: where are the test jars?

File: student-information-management-application/src/main/java/com/student/information/management/data/connection/ConnectionHelper.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package com.student.information.management.data.connection;
```
Comment (RoCS2024) [inline student-information-management-application/src/main/java/com/student/information/management/data/connection/ConnectionHelper.java:1]: Fill in codes for ConnectionHelper

File: student-information-management-application/src/main/java/com/student/information/management/app/facade/student/impl/StudentFacadeImpl.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package com.student.information.management.app.facade.student.impl;
+
+public class StudentFacadeImpl {
```
Comment (RoCS2024) [inline student-information-management-application/src/main/java/com/student/information/management/app/facade/student/impl/StudentFacadeImpl.java:3]: where is implements StudentFacade?

File: student-information-management-application/src/main/java/com/student/information/management/data/student/dao/impl/StudentDaoImpl.java [hunk 5]
```diff
@@ -0,0 +1,4 @@
+package com.student.information.management.data.student.dao.impl;
+
+public class StudentDaoImpl {
```
Comment (RoCS2024) [inline student-information-management-application/src/main/java/com/student/information/management/data/student/dao/impl/StudentDaoImpl.java:3]: where is implements StudentDao?

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comments

**Comment by katieann7** [comment]
Done na po Sir.

**Review by RoCS2024 — APPROVED** [review]

**Review by JustineDaveMagnaye — APPROVED** [review]

**Pull request merged**